content/docs/installation/best-practice.md

```diff
@@ -22,6 +22,120 @@ are designed for backwards compatibility rather than for best practice or maximu
 You may find that the default resources do not comply with the security policy on your Kubernetes cluster
 and in that case you can modify the installation configuration using Helm chart values to override the defaults.
 
+## Network Policy
+
+An example of this recommendation is found in the Calico Documentation:
+> We recommend creating an implicit default deny policy for your Kubernetes pods, regardless if you use Calico or Kubernetes network policy. This ensures that unwanted traffic is denied by default.
+>
+> 📖 [Calico Best practice: implicit default deny policy](https://docs.tigera.io/calico/latest/network-policy/get-started/kubernetes-default-deny#best-practice-implicit-default-deny-policy).
+
```

Comment:
Perhaps an additional link here to the inbuilt `NetworkPolicy` documentation on default policies: https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-policies ?

Reply:
Done.